src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -239,7 +245,7 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
     new JesBackendCall(this, workflowDescriptor, key, locallyQualifiedInputs, abortRegistrationFunction)
   }
 
-  override def engineFunctions: WdlStandardLibraryFunctions = new JesEngineFunctionsWithoutCallContext()
+  override def engineFunctions(interface: IOInterface): WdlStandardLibraryFunctions = new JesEngineFunctionsWithoutCallContext(interface)
```

Comment:
Not that this needs to be addressed now, but we've discussed having the local backend be able to work with GCS.